pom.xml

```diff
@@ -88,56 +87,18 @@
 
         <!-- required plugins -->
         <dependency>
-            <groupId>org.jenkins-ci.plugins.workflow</groupId>
-            <artifactId>workflow-api</artifactId>
+            <groupId>io.jenkins.plugins</groupId>
+            <artifactId>pipeline-groovy-lib</artifactId>
+            <version>999999-SNAPSHOT</version> <!-- TODO -->
```

Comment:
Not possible to get an incremental deployment from the upstream PR until it has been accepted by Hosting.

Reply:
(currently waiting on https://github.com/jenkinsci/pipeline-groovy-lib-plugin/pull/2)